session/bootstrap.go

```diff
@@ -376,6 +376,15 @@ const (
 		column_ids TEXT(19372),
 		PRIMARY KEY (table_id) CLUSTERED
 	);`
+	// CreateStatsHistory stores the historical stats.
+	CreateStatsHistory = `CREATE TABLE IF NOT EXISTS mysql.stats_history (
+		table_id bigint(64) NOT NULL,
+		stats_data longblob NOT NULL,
+		seq_no bigint(64) NOT NULL,
+		version bigint(64) NOT NULL,
+		create_time datetime(6) NOT NULL,
+		PRIMARY KEY(table_id)
```

Comment:
We cannot use `table_id` alone as a primary key here:
```suggestion
UNIQUE KEY table_version (table_id, version)
```